airbyte-integrations/connectors/source-recharge/source_recharge/spec.yaml

```diff
@@ -0,0 +1,32 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/recharge
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Recharge Spec
+  type: object
+  required:
+  - start_date
+  - access_token
+  additionalProperties: true
+  properties:
+    start_date:
+      type: string
+      title: Start Date
+      description: The date from which you'd like to replicate data for Recharge API,
+        in the format YYYY-MM-DDT00:00:00Z. Any data before this date will not be
+        replicated.
+      examples:
+      - '2021-05-14T00:00:00Z'
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+      format: date-time
+    access_token:
+      type: string
+      title: Access Token
+      description: The value of the Access Token generated. See the <a href="https://docs.airbyte.com/integrations/sources/recharge">docs</a>
+        for more information.
+      airbyte_secret: true
+    owner_resource:
+      type: string
+      title: Owner Resource value for metafields resource
+      description: The value will be passed as request option. Possible values include - store, customer or subscription. Default is store, See the <a href="https://developer.rechargepayments.com/2021-01/metafields/metafields_list">docs</a>
+        for more information.
+      default: store
```

Comment:
In the Python connector the `owner_resource` value is not sourced through connector configuration, it is inferred using stream slices/partitions like below. We should implement it the same way here. https://github.com/airbytehq/airbyte/blob/b19ed6d8b398f09b422a888a5a043b964a0a21f9/airbyte-integrations/connectors/source-recharge/source_recharge/api.py#L147-L165